config/kubernetes/sig-release/teams.yaml

```diff
@@ -369,34 +322,22 @@ teams:
         - jeremyrickard # subproject owner / Technical Lead
         - justaugustus # subproject owner / Chair
         - Kasambx # 1.28 Enhancements Shadow
-        - katcosgrove # 1.28 RT Docs Shadow
-        - krol3 # 1.28 Comms Shadow
+        - katcosgrove # 1.28 RT Lead Shadow
```

Comment:
katcosgrove is listed a docs shadow [1], is this line still accurate?
[1] https://github.com/kubernetes/sig-release/blob/master/releases/release-1.28/release-team.md

Reply:
@MadhavJivrajani, fixed in latest commit https://github.com/kubernetes/org/pull/4374/commits/d54330b68455ba4410ad4507c7f4e2a93ee3308e . Thanks!